transformation/bq/test/README.md

```diff
@@ -0,0 +1,16 @@
+# BigQuery integration tests
+
+These are integration tests for quadkey under BigQuery. Divided in 2 categories:
+
+  * Those ending in `_integration.js`. They are integration tests, they use BigQuery so they require authentication. They require `BQ_PROJECTID` and `BQ_DATASET_QUADKEY` environment variables to be defined with the project and dataset where the functions are stored and where tables will be created, and they also require BQ credentials (can be passed in a file using `GOOGLE_APPLICATION_CREDENTIALS` environment variable). Check BIGQUERY.md in the project root for more information on how to set these variables.
```

Comment:
No `BQ_DATASET_QUADKEY`, try finding appearances of the word QUADKEY in your files.